### DISCUSSION TITLE
在Linux平台上产生的临时文件/包按照XDG Base Directory规范放在对应目录里

### DISCUSSION THREAD

**Discussion by 185264646** [xmake-io/xmake#4188] [Ideas]
### 你在什么场景下需要该功能？

[XDG Base Directory Specification (archwiki)](https://wiki.archlinux.org/title/XDG_Base_Directory) 

xmake使用过程中需要远程下载并保存包，会产生一些配置文件，当前这些文件全部被放置于`$HOME/.xmake`目录下。该方案不符合XDG Base Directory规范。实施该方案可以保证项目在linux平台上的可移植性，也可以简化各平台打包所需的卸载逻辑。例如Debian/Ubuntu上的`apt purge`。同时也与较新的linux项目保持一致，如`neovim`，`pip`等。

### 描述可能的解决方案

按照XDG Base Directory规范要求放置xmake产生的临时文件/配置文件/程序包。

[a comment by an automated account is not shown]

**Comment by waruqi** [comment]
不考虑，如果要改，自己配置修改

-  https://xmake.io/#/zh-cn/guide/configuration?id=xmake_globaldir
